quickstarts/microsoft.compute/vm-windows-ssh/main.bicep

```diff
@@ -0,0 +1,264 @@
+@description('Storage Account name')
+param storagename string = 'storage${uniqueString(resourceGroup().id)}'
+
+@description('VM DNS label prefix')
+param vm_dns string = 'winssh-${uniqueString(resourceGroup().id)}'
+
+@description('Admin user for VM')
+param adminUser string
+
+@secure()
+param adminPassword string = newGuid()
+
+@description('Your public SSH key')
+param publicSshKey string
+
+@description('VM size for VM')
+param vmsize string = 'Standard_D4_v5'
```

Comment:
I've submitted a request for quota for V5 series, until that goes through you can use v3